Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.11
+RUN apk add --no-cache coreutils g++ libc-dev cmake make python3
+RUN pip3 install --upgrade pip
+RUN pip3 install junitparser
+WORKDIR /opt/test-runner
+COPY run.sh bin/
```

Comment:
You could consider adding a `.dockerignore` file (see [this example](https://github.com/exercism/csharp-test-runner/blob/master/.dockerignore)) file which speeds up the building of the Docker image. This can definitely be done later.